examples/nextjs-auth-demo/pages/api/me.js

```diff
@@ -1,12 +1,11 @@
 import delay from 'delay';
+import oidc from '../../utils/oidc';
 
-import { getSession } from '../../utils/oidc';
+export default async function me(req, res) {
+  // Introduces a 2.5 sec delay to show client side rendering
+  // await delay(2500);
```

Comment:
```suggestion
  // await new Promise((resolve) => setTimeout(() => resolve(), 2500))
```
We can probably remove this dependency since it's commented out and have the comment show a non-dependency way to do it